packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -110,7 +127,7 @@ export default class AutoUpdaterService implements AutoUpdaterServiceInterface {
 			const release: GitHubRelease = await this.fetchLatestRelease(this.includePreReleases_);
 
 			try {
-				let assetUrl = this.getDownloadUrlForPlatform(release, shim.platformName());
+				let assetUrl = this.getDownloadUrlForPlatform(release, shim.platformName(), shim.architecture());
```

Comment:
Please just use `process.arch` here and remove `shim.architecture` (it sorts of makes sense to have it in the shim but I'm worried it might cause problems on mobile, so let's keep it simple)